tagstudio/src/core/library/alchemy/enums.py

```diff
@@ -42,12 +43,19 @@ class TagColor(enum.IntEnum):
     COOL_GRAY = 36
     OLIVE = 37
 
-    @staticmethod
-    def get_color_from_str(color_name: str) -> "TagColor":
-        for color in TagColor:
-            if color.name == color_name.upper().replace(" ", "_"):
-                return color
-        return TagColor.DEFAULT
+#class TSStandardColors(enum.IntEnum):
```

Comment:
I know you said you aren't a fan of enums, but I think it would be good to have an enum that just has the IDs of colors in the default tagstudio color pack (whatever we will call it).

Reply:
I'll concede to this ;)